cmd/skywire-cli/commands/config/gen.go

```diff
@@ -281,24 +487,18 @@ var genConfigCmd = &cobra.Command{
 			}
 			// create an http client to fetch the services
 			client := http.Client{
-				Timeout: time.Second * 15, // Timeout after 30 seconds
-			}
-			//create the http request
-			req, err := http.NewRequest(http.MethodGet, fmt.Sprint(serviceConfURL), nil)
-			if err != nil {
-				log.WithError(err).Fatal("Failed to create http request\n")
+				Timeout: time.Second * 15, // Timeout after 15 seconds
 			}
-			req.Header.Add("Cache-Control", "no-cache")
-			//check for errors in the response
-			res, err := client.Do(req)
+			// Make the HTTP GET request
+			res, err := client.Get(fmt.Sprint(serviceConfURL))
 			if err != nil {
 				//silence errors for stdout
 				if !isStdout {
 					log.WithError(err).Error("Failed to fetch servers\n")
 					log.Warn("Falling back on hardcoded servers")
 				}
 			} else {
-				// nil error from client.Do(req)
+				// nil error from client.Get
```

Comment:
Can we move all of this code to inside the else clause outside the else clause?  Resp body should be closed even if there is an error I believe. Also, can we set the log level to `panic` or something equivalent and remove all these checks for `!stdout`?

Reply:
valid points, will make the changes

Reply:
I've outdented the code as requested, but in terms of getting rid of 'all these checks for `!isStdout`' it's not just as straightforward as changing the log level because the behavior is also changed by the stdout flag to not create or read files and I already have the -w flag as an override for showing the errors, so it's something I would like to revisit after this PR is merged.

We are talking about code that currently works and is on develop in any case